Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml

```diff
@@ -595,59 +531,1460 @@ tasks:
+
+        Supported Integrations:
+        -Okta
+        -Splunk
+        -QRadar
+        -Azure Log Analytics
+        -PAN-OS
+        -XDR By Palo Alto Networks
+      playbookName: User Investigation - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "185"
+    scriptarguments:
+      AzureSearchTime:
+        simple: ago(3d)
+      OktaSearch:
+        simple: "True"
+      QRadarSearchTime:
+        simple: Last 3 days
+      SIEMFailedLogonSearch:
+        simple: "True"
+      SplunkEarliestTime:
+        simple: -1d
+      SplunkIndex:
+        simple: '*'
+      SplunkLatestTime:
+        simple: now
+      ThreatLogSearch:
+        simple: "True"
+      UserEmail:
+        complex:
+          root: Account
+          accessor: Email
+          transformers:
+          - operator: uniq
+      Username:
+        complex:
+          root: inputs.Username
+          transformers:
+          - operator: uniq
+      XDRAlertSearch:
+        simple: "True"
+      XDRUsernameField:
+        simple: actor_effective_username
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1570,
+          "y": 1660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "140":
+    id: "140"
+    taskid: dcf85ced-cb7b-4ea7-8c5b-e9fa15f92b87
+    type: title
+    task:
+      id: dcf85ced-cb7b-4ea7-8c5b-e9fa15f92b87
+      version: -1
+      name: Investigation - Related Indicators Hunt
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "141"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1780,
+          "y": 1995
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "141":
+    id: "141"
+    taskid: 3468f80b-08b5-45b9-8c94-6d231ef70d20
+    type: condition
+    task:
+      id: 3468f80b-08b5-45b9-8c94-6d231ef70d20
+      version: -1
+      name: Has related Indicators to hunt?
+      description: Checks whether there are any indicators to hunt for.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "152"
+      "yes":
+      - "142"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: RelatedFiles
```

Comment:
Where do we get this key from?

Reply:
these are context keys extracted by the 'TIM - Indicator Relationships Analysis' playbook. should I change the sub-playbooks context to 'Shared globally'?

Reply:
No, if it's in the sub playbook outputs than it's ok as private context 👍